core/src/components/tabbar/tab-button.scss

```diff
@@ -23,6 +23,7 @@
 
   background: transparent;
 
+  text-align: var(--text-align, center);
```

Comment:
Why are we using a fallback value here? Any parent component that sets a different value to `--text-align` will win.

Reply:
Also, text-align is a text property, ie light-dom, do we really need a CSS variable?

We can just use:

```
@include text-inherit();
```

in `.tab-btn` and we can remove a CSS variable!

Reply:
The fallback is so that the text-align is `center` by default. I'm fine with splitting it out and setting it in host, but it wasn't working when set on the parent tabbar iirc.